FRPs/FRP-14.md

```diff
@@ -0,0 +1,31 @@
+---
+id: 14
+title: Cryptoeconomic design exploration
+team: @pdaian @obadiaa 
+created: 2021-06-04
+status: in progress
+---
+
+# Cryptoeconomic design
+
+
+## Background and Problem Statement
+While Flashbots is exploring an SGX-based solution to provide complete pre-trade privacy as outlined in [this post](https://ethresear.ch/t/mev-sgx-a-sealed-bid-mev-auction-design/9677). We would like to explore in parallel if there are cryptoeconomic mechanisms that can reinforce, or substitute the hard cryptography of SGX, to achieve the same desirable result: namely a fully private way to submit tx to block proposers without having to reveal it to them until they cannot tamper with them anymore.
```

Comment:
"While ... . We ..." -> "While ..., we ..."

Reply:
tx ->txs

Reply:
(or "a tx"). see later: "reveal it/them"

Reply:
"until they cannot tampoer with them anymore" is unclear. why "anymore"?

I'd say something like "so that they cannot tamper with them"

Reply:
"until they cannot tampoer with them anymore" is unclear. why "anymore"?

I'd say something like "so that they cannot tamper with them"

Reply:
not sure what you mean by "hard cryptography", but I'd motivate this a bit differently, questioning the need for an expensive, proprietary solution

Reply:
nice, thank you! all great points